### DISCUSSION TITLE
n9e-server and webapi do not start automatically on boot

### DISCUSSION THREAD

**Discussion by coolmax123** [ccfos/nightingale#1425] [Q&A]
操作系统：Centos7.9
N9E版本：n9e-v5.15.0-linux-amd64.tar.gz

通过systemctl start n9e-server.service和systemctl start n9e-webapi.service手动是能正常running，但是重启服务器后无法自启，报错信息如下：
![图片](https://user-images.githubusercontent.com/127930630/225240128-034a0e76-c882-4ce7-89f3-ab15acca2b12.png)
![图片](https://user-images.githubusercontent.com/127930630/225240227-1c2743ca-843e-4bde-a67f-b5794004abdc.png)

权限如下：
![图片](https://user-images.githubusercontent.com/127930630/225239975-de90cecd-8e91-4a81-9808-cebfce9a2ffa.png)
请大神看下如何解决？

n9e-server和n9e-webapi的启动文件如下：
**配置n9e-server启动文件**
vi /etc/systemd/system/n9e-server.service
[Unit]
Description="n9e-server"
After=network.target

[Service]
Type=simple
ExecStart=/opt/n9e/n9e server
WorkingDirectory=/opt/n9e
Restart=on-failure
SuccessExitStatus=0
LimitNOFILE=65536
StandardOutput=syslog
StandardError=syslog
SyslogIdentifier=n9e-server

[Install]
WantedBy=multi-user.target

**配置n9e-webapi启动文件**
vi /etc/systemd/system/n9e-webapi.service
[Unit]
Description="n9e-webapi"
After=network.target

[Service]
Type=simple
ExecStart=/opt/n9e/n9e webapi
WorkingDirectory=/opt/n9e
Restart=on-failure
SuccessExitStatus=0
LimitNOFILE=65536
StandardOutput=syslog
StandardError=syslog
SyslogIdentifier=n9e-webapi

[Install]
WantedBy=multi-user.target

查询log
![图片](https://user-images.githubusercontent.com/127930630/225255940-b5f0e1c8-84a5-4412-b494-23fb038cf2b9.png)
3306服务是启动的
![图片](https://user-images.githubusercontent.com/127930630/225256076-9cb7e536-c494-4b5b-9ea9-8f6d8a3b2c8c.png)

**Comment by UlricQin** [comment]
程序不会骗人，dial tcp 127.0.0.1:3306 connection refused 就是连不上，至于为啥连不上你得自己查了，可以尝试改一下mysql的bind-address之类的，还有firewalld之类的，selinux之类的
